cabfile/example_test.go

```diff
@@ -0,0 +1,32 @@
+package cabfile
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	cabfile "github.com/google/go-cabfile/cabfile"
+)
+
+func ExampleNext() {
```

Comment:
This file should follow the guidance in https://go.dev/doc/tutorial/add-a-test - i.e. use testing as the runtime and take a *testing.T for reporting failures. Also Data1.cab is missing, so this test won't be reproducible for others.